src/reducers/tasksReducer.js

```diff
@@ -135,16 +136,38 @@ export default function(state = initialState, action) {
       });
     }
 
-    case 'HYPOTHESIS_GROUP_CREATE': {
-      console.log('HYPOTHESIS_GROUP_CREATE reducer called...');
-      const { taskGid, name, description } = action;
-      // TODO(elimisteve): Use name, description to actually create
-      // Hypothesis group; faking it by updating local state
+    case 'HYPOTHESIS_GROUP_CREATING': {
+      // simply set the task as in progress of creating a hypothesis group
+      const { taskGid } = action;
+      const task = state.taskMap[taskGid];
+
+      return Object.assign({}, state, {
+        taskMap: Object.assign({}, state.taskMap, {
+          [taskGid]: Object.assign({}, task, {
+            creatingHypothesisGroup: 'in progress'
+          })
+        })
+      });
+    }
+
+    case 'HYPOTHESIS_GROUP_CREATED': {
+      // assign the data returned from hypothesis call to the task
+      const { taskGid, hypothesisData } = action;
       const task = state.taskMap[taskGid];
+
       return Object.assign({}, state, {
         taskMap: Object.assign({}, state.taskMap, {
           [taskGid]: Object.assign({}, task, {
-            deliverables: task.deliverables += '\n\n#### Hypothesis Group\n\n<https://hypothes.is/groups/qodKXB7q/p-juansanchez>'
+            creatingHypothesisGroup: 'done',
+            deliverables: (task.deliverables +=
+              '* Hypothesis Group: ' +
+              '[' +
+              hypothesisData.name +
+              '](' +
+              hypothesisData.links.html +
+              ' "' +
+              hypothesisData.name +
+              '")')
```

Comment:
Use backticks here: "* Hypothesis Group: [${hypothesisData.name}](hypothesisData.links.html) ... "

Reply:
linter is still splitting this in a weird way, now it's just one place though.
Any way we can prevent it from splitting it like: 
    deliverables: (task.deliverables += `* Hypothesis Group: [${
        hypothesisData.name
    }](${hypothesisData.links.html} "${hypothesisData.name}")`)